Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -282,8 +297,10 @@ public struct WorldScaleSceneView: View {
         }
         return try await ARGeoTrackingConfiguration.checkAvailability()
     }
+#endif
 }
 
+@available(visionOS, unavailable)
```

Comment:
I suggest putting this on the type itself rather than on the extension.

Reply:
It is already on `WorldScaleSceneView` and all extensions of that type will need it as well.